descope/auth/types.go

```diff
@@ -25,6 +25,7 @@ type AuthenticationInfo struct {
 type WebAuthnTransactionResponse struct {
 	TransactionID string `json:"transactionId,omitempty"`
 	Options       string `json:"options,omitempty"`
+	Create        bool   `json:"create,omitempty"`
```

Comment:
I think we should split `WebAuthnTransactionResponse` to `WebAuthnSignUpOrInTransactionResponse` that will embed `WebAuthnTransactionResponse`, the api should not expose that sign up (or in) can return `Create`